.gitlab/quartz-build-and-test.yml

```diff
@@ -23,7 +23,7 @@ allocate_resource:
   stage: allocate_resource
   script:
     - echo ${ALLOC_NAME}
-    - salloc --exclusive --nodes=1 --partition=pdebug --time=45 --no-shell --job-name=${ALLOC_NAME}
+    - salloc --exclusive --nodes=1 --partition=pdebug --time=60 --no-shell --job-name=${ALLOC_NAME}
```

Comment:
I suggest that you switch to the new CI queue instead of using the pdebug one:
Replace `--partition=pdebug` with ` --reservation=ci`.

Reply:
Oops, this is quartz. I should check if Quartz has the CI reservation...

Reply:
Yes it does.

Reply:
@tzanio, do you want to make this change?

Reply:
Done